custom/ewsghana/reports/__init__.py

```diff
@@ -287,7 +288,7 @@ def report_context(self):
 
     def get_report_context(self, data_provider):
         total_row = []
-        headers = []
+        headers = DataTablesHeader()
```

Comment:
Does this mean that there are some reports that have a blank `headers` value, and if so is that expected?